tapdb/sqlc/migrations/000007_universe.up.sql

```diff
@@ -81,3 +81,29 @@ CREATE VIEW universe_stats AS
     FROM universe_events u
     JOIN universe_roots roots ON u.universe_root_id = roots.id
     GROUP BY roots.asset_id, roots.group_key, roots.namespace_root;
+
+-- This table contains general configuration for universe federation syncing.
+CREATE TABLE IF NOT EXISTS federation_general_sync_config(
+    -- This ID column is used to ensure that only one row exists in this table.
+    id INTEGER PRIMARY KEY,
+
+    -- This flag indicates that, in general, only issuance proofs should be
+    -- synced.
+    only_sync_issuance_proofs BOOLEAN NOT NULL
```

Comment:
IMO this is a bit overkill for a single boolean value... Should we perhaps instead create a generic `config` table where we can have key/value (and maybe a field that indicates the type to parse the value into)? That way we don't need to create new tables for any new config items we want to store.

Reply:
Please say more about the generic config table you have in mind.

What would be the types of the key and value columns?

How would we differentiate general config from universe specific?

Reply:
Just something like this:
```
create table config (
key text not null,
value text not null
)
```

And thinking about it, we probably don't even need to add the type, the caller would know what type of value to expect.
Example usage would be:
```
tapdb.StoreConfigBool("federation.sync.only_issuance_proofs", true) // would convert the value to `"true"`
tapdb.StoreConfigString("foo.bar", "myvalue")

myBool := tapdb.LoadConfigBool("universe.sync.only_issuance_proofs") // would use `strconv.ParseBool()`
myString := tapdb.LoadConfigString("foo.bar")
```

That would just replace the single boolean value in `federation_general_sync_config` but give us a way to store future config values more easily.

For universe specific stuff like `federation_uni_sync_config` we'd probably still need a table.
But yeah, maybe it's total overkill for now, was just an idea.

Reply:
Or the simple alternative for this specific case would be to only have the `federation_uni_sync_config` table but allow the `id` field to be `NULL` which would then represent the general config.

Reply:
Wouldn't that config table also need some sort of name spacing? Though I _do_ see value in having a nice abstraction of persistent configuration. Not sure where else we'd use it at this point though. Arguably, the other config items like the federation members could also use this. IMO ultimately nice to have some specific structure though. 

So is this a case of YAGNI given we have a set use case for now?